network-api/networkapi/buyersguide/factory.py

```diff
@@ -92,33 +96,33 @@ def generate(seed):
 
     print('Generating fixed Buyer\'s Guide Product for visual regression testing')
     ProductFactory.create(
-        product_words=['Percy', 'Cypress'],
-        name='percy cypress',
-        draft=False,
         adult_content=False,
-        company='Percy',
         blurb='Visual Regression Testing',
-        url='https://vrt.example.com',
-        price=350,
         camera_app=True,
-        meets_minimum_security_standards=True,
         camera_device=False,
-        microphone_app=True,
-        microphone_device=False,
+        child_rules=False,
```

Comment:
This was due to telling VSCode to "sort all lines ascending" because I had to dig through the field names and this was way easier to find the fields I had to change =)